src/Components/Modal/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react'
+import { ModalProps } from './models'
+import './styles.scss'
+
+const Modal: React.FC<ModalProps> = ({ isOpen, closeAction, Content }) => {
```

Comment:
why don't you use `children` instead of `Content`?

Reply:
Just want try new approach, but you right, children more predictable :)
Fixed.